doc/changelog.rst

```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
 SmartSim
 ========
 
+0.4.1
+-----
+
+Released on June 24, 2022
+
+Description:
+This release of SmartSim introduces a new experimental feature to help make
+SmartSim workflows more portable: the ability to run simulations models in a
+container via Singularity. This feature has been tested on a small number of
+platforms and we encourage users to provide feedback on its use.
+
+We have also made incremental improvements in a variety of areas: being able to
+load scripts and machine learning models into the database directly from
+SmartSim driver scripts, the ability to use `KeyDB` instead of `redis-server`,
```

Comment:
Maybe phrase as "the ability to choose `KeyDB` or `Redis` during SmartSim installation"

Reply:
Slightly reworded from your suggestion to "install-time choice to use either `KeyDB` or `Redis` for the Orchestrator"